### PR TITLE
#1221: bump okhttp3 version from 3.14.6 to 4.6.0

### PR DESCRIPTION
**Pull request by Sam-Kruglov**
See #1221

### CODE REVIEW

File: okhttp/src/main/java/feign/okhttp/OkHttpClient.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ static Request toOkHttpRequest(feign.Request input) {
       }
     }
 
-    RequestBody body = inputBody != null ? RequestBody.create(mediaType, inputBody) : null;
+    RequestBody body = inputBody != null ? RequestBody.create(inputBody, mediaType) : null;
```
Comment (Sam-Kruglov) [inline okhttp/src/main/java/feign/okhttp/OkHttpClient.java:87]: the previous signature is deprecated

Comment (velo) [reply okhttp/src/main/java/feign/okhttp/OkHttpClient.java:87]: I would say to use the deprecated one, so we keep feign compatible with a wider number or releases

**Comment by Sam-Kruglov** [comment]
Thanks, @velo! Can we merge it?

**Comment by velo** [comment]
> Thanks, @velo! Can we merge it?

Will do, as soon I get to a computer... Still 6am here :)

**Pull request merged**